packages/spacecat-shared-gpt-client/README.md

```diff
@@ -58,7 +60,8 @@ async function fetchInsights(prompt) {
       log: console,
     });
 
-    const insights = await client.fetch(prompt);
+    // Internally, use Firefall's .../capability_execution/job endpoint.
```

Comment:
Almost talked myself out of this comment since it is an implementation detail, but thought it might help some dev's with Firefall experience.